config/test/kubemacpool.yaml

```diff
@@ -229,6 +219,59 @@ spec:
 ---
 apiVersion: apps/v1
 kind: Deployment
+metadata:
+  labels:
+    control-plane: cert-manager
+    controller-tools.k8s.io: "1.0"
+  name: kubemacpool-cert-manager
+  namespace: kubemacpool-system
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      control-plane: cert-manager
+      controller-tools.k8s.io: "1.0"
+  strategy:
+    type: Recreate
+  template:
+    metadata:
+      labels:
+        app: kubemacpool
+        control-plane: cert-manager
+        controller-tools.k8s.io: "1.0"
+    spec:
+      containers:
+      - args:
+        - --v=production
```

Comment:
I think you should add an entry to the [kustomize](https://github.com/k8snetworkplumbingwg/kubemacpool/blob/master/config/test/kustomization.yaml#L9) file in tests folder so that this will be changed to `debug` like the `kubemacpool-mac-controller-manager` pod.

Reply:
Done